examples/animate-d3-with-mixin/index.js

```diff
@@ -0,0 +1,156 @@
+var React = require('react')
+var ReactDOM = require('react-dom')
+var Faux = require('../../lib/ReactFauxDOM')
+var d3 = require('d3')
+
+var Chart = React.createClass({
+  mixins: [Faux.mixin],
+  getInitialState: function () {
+    return { look: 'stacked' }
+  },
+  render: function () {
+    return <div>
+      <button onClick={this.toggle}>Toggle</button>
+      {this.state.chart}
+    </div>
+  },
+  toggle: function () {
+    if (this.state.look === 'stacked') {
+      this.setState({ look: 'grouped' })
+      this.transitionGrouped()
+    } else {
+      this.setState({ look: 'stacked' })
+      this.transitionStacked()
+    }
+  },
+  componentDidMount: function () {
+    // This will create a faux div and store its virtual DOM
+    // in state.chart
+    var faux = this.connectFauxDOM('div', 'chart')
```

Comment:
Unless I'm confusing myself, I do mean virtual DOM. Although the faux nodes are stored in the `this.connectedFauxDOM` object, only the `fauxnode.toReact()` result is ever stored in state (that is, sent to `this.setState). Or perhaps I'm misusing the label here? In my mind virtual DOM is the sketetal representative object structure mirroring the DOM, which is what `fauxnode.toReact` outputs, right?